zerver/lib/integrations.py

```diff
@@ -103,6 +103,7 @@ def __init__(self, name, display_name=None, logo=None, logo_alt=None, git_url=No
     WebhookIntegration('freshdesk'),
     WebhookIntegration(
         'github',
+        url='api/v1/external/deprecated_github',
```

Comment:
I'm not sure about URLs, should we have 3 urls:
`/github`
`/webhook_github`
`/deprecated_github`
or just first one?